### PR TITLE
Save system load during export logs to have idea about loaded systems

### PR DESCRIPTION
**Pull request by okurz**
Related issue: https://progress.opensuse.org/issues/12064

### CODE REVIEW

**Comment by nilxam** [comment]
LGTM

**Comment by Soulofdestiny** [comment]
LGTM, like the idea

File: tests/console/consoletest_setup.pm [hunk 1]
```diff
@@ -60,8 +60,9 @@ sub run() {
     select_console 'root-console';
 
     type_string "chown $username /dev/$serialdev\n";
-    # Export the existing status of running tasks for future reference (fail would export it again)
+    # Export the existing status of running tasks and system load for future reference (fail would export it again)
     script_run "ps axf > /tmp/psaxf.log";
+    script_run "cat /proc/loadavg > /tmp/loadavg_consoletest_setup";
```
Comment (coolo) [inline tests/console/consoletest_setup.pm]: no need to duplicate the test name in the file - upload_logs already does that.

Comment (okurz) [inline tests/console/consoletest_setup.pm]: it was more like lack of fantasy to come up with a better name, e.g. could have been "loadavg_before_any_potential_later_fail_to_which_we_want_to_compare"

**Pull request merged**